src/main/java/org/fundacionjala/pivotal/common/CommonMethods.java

```diff
@@ -0,0 +1,40 @@
+package org.fundacionjala.pivotal.common;
+
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.WebDriverWait;
+
+/**
+ * Class CommonMethods.
+ */
+public final class CommonMethods {
+
+    /**
+     * Method for init Driver wait.
+     */
+    private static WebDriverWait DRIVER_WAIT = WebCommonDriver.getWebCommonDriver().getDriverWait();
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Rename this field "DRIVER_WAIT" to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS3008)